### PR TITLE
Feat/sidebar

### PR DESCRIPTION
**Pull request by reganlosey**
What (if any) features are you implementing?

- Set up page's sidebar, add colors and borders.
- 
What (if anything) did you refactor?
-
Were there any issues that arose?
- Let's look into flexbox tomorrow.
- 
Is there anything that you need from your teammate?

- Let's get together and try to get this positioning ironed out.
- 
Any other comments, questions, or concerns?

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -1,9 +1,33 @@
...
+  font-family: 'Open Sans';
+}
+/* .container{
+  display: flex;
+} */
+
+.filter-starred-ideas {
+  border: 2px solid white;
+}
+.sidebar {
```
Comment (edwardkrupicka) [inline styles.css:11]: Looks good so far, maybe we should try to use flex box in here somehow. I'll do some research

**Review by edwardkrupicka — COMMENTED** [review]
Looks great, we have a lot to work with here!

File: styles.css [hunk 2]
```diff
@@ -1,9 +1,33 @@
...
-    font-family: 'Open Sans', Bold;
+    font-family: 'Open Sans';
+    font-weight: 100;
     color: #1F1F3D;
     background-color: #A9AAD2;
     border: 1px solid black;
+    margin-left: 300px;
 }
 
 .main-input-box {
```
Comment (ivonne-hernandez) [inline styles.css:33]: Was this deleted? Or just a placeholder for when we figure out what needs to go in here?

File: styles.css [hunk 3]
```diff
@@ -32,7 +56,7 @@
 }
 
 .idea-grid {
-  background-color: EAEAF4
+  background-color: #EAEAF4;
   display: grid;
   grid-template-columns: 1fr 7fr;
```
Comment (ivonne-hernandez) [inline styles.css:62]: We can delete this empty line.

**Review by ivonne-hernandez — APPROVED** [review]
Please review the feedback that was provided for each individual file.

**Pull request merged**